src/configs/index.js

```diff
@@ -1 +1,57 @@
 export const LANGUAGE = 0;
+
+export const DARK_THEME = 'dark';
+export const LIGHT_THEME = 'light';
+
+export const HOMEPAGE_LOOKS_IMAGES = [
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img1.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img2.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img3.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img4.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img5.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img6.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img7.jpg',
+  'https://horondi.blob.core.windows.net/horondi/our-looks/img8.jpg'
+];
+
+export const CABINET_OPTIONS_NOT_LOGGED = {
+  0: {
+    wishlist: 'Список уподобань',
+    changeTheme: 'Змінити тему',
+    logIn: 'Увійти'
+  },
+  1: {
+    wishlist: 'Wishlist',
+    changeTheme: 'Change theme',
+    logIn: 'Log in'
+  }
+};
+
+export const CABINET_OPTIONS_LOGGED = {
+  0: {
+    profile: 'Профіль',
+    wishlist: 'Список уподобань',
+    changeTheme: 'Змінити тему',
+    logOut: 'Вийти'
+  },
+  1: {
+    profile: 'Profile',
+    wishlist: 'Wishlist',
+    changeTheme: 'Change theme',
+    logOut: 'Log out'
+  }
+};
+
+export const HOMEPAGE_TITLES = {
+  0: {
+    catalog: 'Каталог',
+    look: 'Стиль Горонді'
+  },
+  1: {
+    catalog: 'Catalog',
+    look: 'Horondi style'
+  }
+};
+
+export const LOGO = 'HORONDI';
+export const URL_LANGUAGE = 'en';
```

Comment:
why do you need this variable if you use LANGUAGE constant

Reply:
i should to ask @ecl1pseo for this